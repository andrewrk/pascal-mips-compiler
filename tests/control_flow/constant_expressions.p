program Main;
class Main begin
    function Main;
        var a : Integer;
        var b : Integer;
        var c : Boolean;
        var d : Boolean;
    begin
        a := (1 + 2) * 3 / (4 - 5) * 2 + 42 + 1337;
        print a;
    end
end
.

