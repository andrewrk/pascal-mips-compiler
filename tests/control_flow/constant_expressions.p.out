-1277
7
-1277
12
